packages/protoc-gen-connect-query/snapshots/buf.gen.yaml

```diff
@@ -0,0 +1,22 @@
+# buf.gen.yaml defines a local generation template.
+# For details, see https://docs.buf.build/configuration/v1/buf-gen-yaml
+version: v1
+plugins:
+  - name: es
+    path: ../node_modules/.bin/protoc-gen-es
```

Comment:
This line can be removed. 

```suggestion
```

For context, with `plugin: es`, the buf CLI tries to use `protoc-gen-connect-es` on your $PATH. 

If we're calling `buf generate` from an npm script or with `npx` or `pnpm exec`, it's run with a modified $PATH that includes the npm "binaries".

In general, we should use the simple form and avoid the full node_modules path. I don't think we can do it for protoc-gen-connect-query _inside_ of this package, because pnpm does not wire it up (it doesn't depend on itself). 

But we can and IMO should use the simple form in the example. If you think it's too much magic, I'd still prefer we use the simple form, but explain it in a comment.

Reply:
No that makes sense, i'll adjust.